sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/tests/Tests/AppCRUDTests.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.SpringAppDiscovery.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.SpringAppDiscovery.Tests.Tests
```

Comment:
we should not have two levels of `Tests` in the namespace
```suggestion
namespace Azure.ResourceManager.SpringAppDiscovery.Tests
```